static/js/remarkable.js

```diff
@@ -50,16 +50,38 @@ let lastY = null;
 
 let penColor = "rgb(0, 0, 0)";
 
+// Don't write with tip when paused
+let pause = false;
 
+// The 'glow' for pen, eraser and pause/pointer
+const overlayColors = {
+    "PEN": "rgb(200, 200, 255)",
+    "ERASER": "rgb(255, 150, 150)",
+    "POINTER": "rgb(0, 150, 80)",
+  }
+let overlayPenWidth = 5
+let overlayEraserWidth = 10 // also defines actual eraser width
+
+// pen icon and handler for tool changes
 pen_menu_logo = document.getElementById("pen")
 pen_menu_logo.addEventListener('load', function () {
-    if (last_tool == tools.TIP && penColor != pencolors.BLACK) {
-        pen_menu_logo.contentDocument.getElementById("icon").setAttribute("fill", penColor);
-    } else {
-        pen_menu_logo.contentDocument.getElementById("icon").setAttribute("fill", "#efeff3");
-    }
+    setPenIconStyle();
 });
 
+function setPenIconStyle() {
+    if (pause) {
+        // grey pen icon in pause mode
+        pen_menu_logo.contentDocument.getElementById("icon").setAttribute("fill", pencolors.GRAY);
+    } else {
+        if (last_tool == tools.TIP && penColor != pencolors.BLACK) {
+            // coloured icon if coloured pen
+            pen_menu_logo.contentDocument.getElementById("icon").setAttribute("fill", penColor);
+        } else {
+            // light grey icon if eraser or black pen
+            pen_menu_logo.contentDocument.getElementById("icon").setAttribute("fill", pencolors.LIGHTGRAY);
+        }
+    }
+}
```

Comment:
To an `utils.js`

Reply:
Sure. It's your project :) Shall I or would you like to do it?

Reply:
I'll do it.